apps/web/src/components/AccessRisk/AccessRiskTooltips.tsx

```diff
@@ -47,12 +50,14 @@ const AccessRiskTooltips: React.FC<AccessRiskTooltipsProps> = ({
         <Link style={{ display: 'inline' }} ml="4px" external href="https://www.hashdit.io">
           HashDit
         </Link>
-        <Flex mt="4px">
-          <Text>{t('Get more details from')}</Text>
-          <Link ml="4px" external href={`https://dappbay.bnbchain.org/risk-scanner/${tokenAddress}`}>
-            {t('RedAlarm')}
-          </Link>
-        </Flex>
+        {chainId === ChainId.BSC && (
+          <Flex mt="4px">
+            <Text>{t('Get more details from')}</Text>
+            <Link ml="4px" external href={`https://dappbay.bnbchain.org/risk-scanner/${tokenAddress}`}>
```

Comment:
can not see risk info from this url: https://dappbay.bnbchain.org/risk-scanner/0xa80221D067603e30060F75E2458AA361f8eE5402

not sure if is network problem

Reply:
looks ok on my end

Reply:
same looks ok on my end